pom.xml

```diff
@@ -227,6 +227,29 @@ language governing permissions and limitations under the License. -->
                 <version>1.5.3-M1</version>
             </dependency>
             <dependency>
+                <groupId>org.apache.ignite</groupId>
+                <artifactId>ignite-core</artifactId>
+                <version>1.6.0</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.ignite</groupId>
+                <artifactId>ignite-spring</artifactId>
+                <version>1.6.0</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apached.ignite</groupId>
+                <artifactId>ignite-core</artifactId>
+                <version>1.6.0</version>
+                <type>test-jar</type>
+                <scope>test</scope>
```

Comment:
I'd suggest against providing the scope here and instead putting it in the pom where it's used. Adding the scope test here will propagate to any package that brings in this dependency (even transitively).

Comment also applies to ignite-log4j2